### ISSUE TITLE
Wrong CSS prefix for content element

### ISSUE THREAD

**Issue by fritzmg** [netzmacht/contao-leaflet-maps#90]
Just a small issue I noticed: if you use the content element `leaflet`, which will use the default template `ce_leaflet_map`, its CSS class will be `mod_leaflet` instead of `ce_leaflet`.

**Comment by dmolineus** [comment]
This should be fixed with contao-toolkit version 3.4.0. which one is installed in your application?

**Comment by fritzmg** [comment]
```
$ composer show netzmacht/contao-toolkit
name     : netzmacht/contao-toolkit
descrip. : This library provides tools to simplify common tasks when developing with Contao.
keywords : contao, dev, tools
versions : * 3.4.2
```

**Comment by dmolineus** [comment]
Will be fixed through netzmacht/contao-toolkit 3.4.5, see https://github.com/netzmacht/contao-toolkit/commit/2e5b530b12c3496cc57d6626d4dd0a011bbca786
